src/library/Menu/styled.js

```diff
@@ -32,32 +27,11 @@ export const MenuDividerRoot = createStyledComponent(
   }
 );
 
-export const MenuGroupRoot = createStyledComponent(
-  'div',
-  (props) => {
-    const theme = menuGroupTheme(props.theme);
-
-    return {
```

Comment:
It doesn't seem like these styles are represented in the changes?

Reply:
I think the `:first-child` bit was unaccounted for. Updated.